package.json

```diff
@@ -150,9 +151,9 @@
         "react-dom": "16.14.0",
         "react-icons": "3.11.0",
         "react-router-dom": "6.9.0",
+        "react-test-renderer": "16.14.0",
```

Comment:
not sure why this change happens in this PR, but since it is here: do we need this package at all?